src/components/ChangelogDialog/components/Picture/Picture.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {Loader} from '../../../Loader';
+
+import './Picture.scss';
+
+const b = block('changelog-dialog-picture');
+
+type LoadingState = 'loading' | 'loaded' | 'error';
+
+export interface PictureProps {
+    className?: string;
+    src: string;
+    ratio: number;
+}
+
+export function Picture({className, src, ratio}: PictureProps) {
+    const [loadingState, setLoadingState] = React.useState<LoadingState>('loading');
+
+    React.useEffect(() => {
+        setLoadingState('loading');
+
+        const img = new Image();
```

Comment:
Not available in SSR

Reply:
Added check for `typeof window === 'object'`

Reply:
`useEffect` does not run during SSR

Reply:
Yep, removed the check